packages/lesswrong/components/ea-forum/giving-portal/EAGivingPortalPage.tsx

```diff
@@ -302,19 +302,22 @@ const EAGivingPortalPage = ({classes}: {classes: ClassesType}) => {
   const [notifyForVotingOn, setNotifyForVotingOn] = useState(currentUser?.givingSeasonNotifyForVoting ?? false);
   const {flash} = useMessages();
   const {openDialog} = useDialog();
+  const { captureEvent } = useTracking({ eventProps: { pageContext: "eaGivingPortal" } });
 
   const toggleNotifyWhenVotingOpens = useCallback(() => {
-    // TODO: captureEvent
+    captureEvent('toggleNotifyWhenVotingOpens', { notifyForVotingOn: !notifyForVotingOn })
+
     if (!currentUser) {
       openDialog({
         componentName: "LoginPopup",
         componentProps: {},
       })
+      return;
```

Comment:
Unrelated: fix for this toggle changing when logged out (https://app.asana.com/0/1205806305763861/1205853507203268/f)